### ISSUE TITLE
Always app is asking for admin consent for normal users

### ISSUE THREAD

**Issue by ymihir** [Azure-Samples/active-directory-dotnet-admin-restricted-scopes-v2#4]
I am using `User.Read Mail.Send Files.ReadWrite Group.ReadWrite.All` permissions to do some operations using groups. Once I open the app it is asking for admin consent. I have logged in as admin and given permissions for the app. Now I have logged in as normal users. At this point app is asking for again admin consent. What wrong I am doing here?
Once I approve the app with admin will it be not applicable to every user?
I am storing above credentials in web.config file.

- [x] Question

**Comment by ymihir** [comment]
May be somewhere I am doing wrong. I have created app under https://apps.dev.microsoft.com/ and given permissions here. Also in my web.config file I have given permissions `User.Read Mail.Send Files.ReadWrite Group.ReadWrite.All`. So when I run the app it is asking for admin consent. I have logged in as admin it is not asking me any consent (may be i have approved earlier), if I login as normal user then still it is asking me again admin consent. 
Also I have approved the permissions by accessing below url: https://login.microsoftonline.com/common/adminconsent?client_id=75acb700-e675-4dc4-9deb-d3f68f6911b2&state=12345&redirect_uri=http://localhost:34280/
